File: Lab2/Lab_2_3.cpp
#include <iostream>
#include <vector>

using namespace std;

int Turn(int n) {
	int m = 0;
	while (n)
	{
		m = m * 10 + n % 10;
		n /= 10;
	}
	return m;
}


int main() {
	int num = 0;
	int a = 0;
	vector <int> myvec;
	
	cout << "Enter amount of numbers:";
	cin >> num;

	for (int i = 0; i < num; i++) {
		cout << endl << "Enter " << i+1 << " numb:";
		cin >> a;
		a = Turn(a);
		myvec.push_back(a);
		cout << endl;
	}
	for (int i = 0; i < num; i++) {
		cout << myvec[i] << endl;
	}
	

	return 0;
}

File: Lab2/Lab_2_3.py
print('Enter count of numbers:')
n = int (input())
A = []
def turn(n1):
    n2 = 0
    while n1 > 0:
        digit = n1 % 10
        n1 = n1 // 10
        n2 = n2 * 10
        n2 = n2 + digit 
    return n2
    
A += [int(input('Enter A[%d] = '%(i+1))) for i in range(0,n)]
for i in range (0,n):
    A[i]=turn(A[i])
    
for i in range(0,n):
    print(A[i])
